cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1317,11 +1317,32 @@ declare namespace Deno {
    */
   export function openKv(path?: string): Promise<Deno.Kv>;
 
+  /** **UNSTABLE**: New CronSchedule interface.
+   * @category cron
+   */
+  type CronScheduleExpression = number | { exact: number | number[] } | {
+    start?: number;
+    end?: number;
+    every?: number;
+  };
+  
+  export interface CronSchedule {
```

Comment:
This also needs jsdoc with description, `**UNSTABLE**`, and `@category Cron`

Reply:
Added decription.